05_sumAll/solution/sumAll-solution.js

```diff
@@ -1,13 +1,10 @@
 const sumAll = function (min, max) {
   if (!Number.isInteger(min) || !Number.isInteger(max)) return "ERROR";
   if (min < 0 || max < 0) return "ERROR";
-  if (min > max) {
-    const temp = min;
-    min = max;
-    max = temp;
-  }
+  if (min > max) [min, max] = [max, min];
+
```

Comment:
Not entirely sure if we should use this as the TOP solution, just because I don't believe it's something introduced to users at the point they're asked to do this exercise (believe it's the lesson they're first introduced to arrays).

Maybe we could add a comment that shows this method as an alternative.

Reply:
Done